tools/src/main/java/org/apache/kafka/tools/consumer/group/ShareGroupCommandOptions.java

```diff
@@ -159,11 +159,11 @@ public void checkArgs() {
         if (options.has(describeOpt)) {
             if (!options.has(groupOpt) && !options.has(allGroupsOpt))
                 CommandLineUtils.printUsageAndExit(parser,
-                    "Option " + describeOpt + " takes one of these options: " + allGroupSelectionScopeOpts.stream().map(Object::toString).collect(Collectors.joining(", ")));
+                    "Option " + describeOpt + " takes one of these options: " + allGroupSelectionScopeOpts.stream().map(Object::toString).sorted().collect(Collectors.joining(", ")));
```

Comment:
What about using `LinkedHashSet` when group of options are created and avoid repeated sorting?